MainApp/src/main/res/layout/touchpad_container.xml

```diff
@@ -38,9 +38,7 @@
                 android:textColor="#FFFFFF"
                 android:onClick="clickDial"
                 android:text="@string/dial_button"
-                android:paddingStart="40dp"
-                android:paddingEnd="40dp"
```

Comment:
This was done to increase the width of the dial button, so it's a larger target for pressing. Without it, the button can be quite small, especially on larger devices.

Using Start/End instead of Left/Right was simply automatic on my part. Kind of trained myself to be i18n-aware and use that everywhere, but in this case it doesn't specifically mean anything, since it's equal on both sides.

However! It did draw attention to the fact that Froyo completely ignores Start/End. So I've changed this to Left/Right and everyone, Froyo on up, is happy. Something to keep in mind. :+1:  :)

(Additionally, while testing my assumptions in a Froyo emulator, I noticed several other bugs I didn't know about. Fixes incoming!)